### ISSUE TITLE
Adding the Roach

### ISSUE THREAD

**Issue by Danimal696** [OpenDungeons/OpenDungeons#962]
My most complete rig and animation to date, but the Roach is finally here after many hours of animation work:
https://github.com/OpenDungeons/OpenDungeons/pull/961

Shoulndt conflict with other PR´s, needs an entry to be created in
config file once the Creature skill PR is merged; spider one can be
copied but mesh size needs to be 0.15 in all 3 values.

![roach2](https://cloud.githubusercontent.com/assets/8299756/10123808/4c4a0296-6543-11e5-8b0d-27e4567ec823.png)

**Comment by hwoarangmy** [comment]
looks good. I will integrate it when the Creature skill PR is in if it is not already done ^^

**Comment by Bertram25** [comment]
Cool :D

**Comment by hwoarangmy** [comment]
I've had a look and it seems pretty nice. Congrats ^^

**Comment by hwoarangmy** [comment]
Note that this is closed by #961

**Comment by Danimal696** [comment]
one less to go! :D
